codecov.yml

```diff
@@ -0,0 +1,15 @@
+comment:
+  layout: header, changes, diff, sunburst
+coverage:
+  status:
+    patch: false
+    # project will give us the diff in the total code coverage between a commit
+    # and its parent
+    project:
+      default:
+        target: auto
+        threshold: "0.05%"
+    changes: false
+ignore:
+  - "**/internal/test"
+  - "vendor/*"
```

Comment:
no mocks for now I think. no integration test either (all are unit tests). `internal/test` is for some utils code (`builders`, …) for the tests, that's why I ignored them 👼